### PR TITLE
Show summary of changes in v1.4 on the default index page

### PR DESCRIPTION
**Pull request by Gazler**
I thought this might be a good idea for the main things that have been added/changed that people may not know about.

This may just be a maintenance burden/more for people to delete when they generate a project though, so am happy for any feedback.

Here's a screenshot:

phoenix-changes

### CODE REVIEW

File: installer/templates/phx_web/templates/page/index.html.eex [hunk 1]
```diff
@@ -3,6 +3,49 @@
...
+          <li>
+            <a href="https://github.com/phoenixframework/phoenix/pull/2801">
+              <code>Router.Helpers.static_push/2</code> function for HTTP/2 server push
+          </li>
+          </li>
+        </ul>
+      </li>
+      <li>
+        <a href="https://github.com/phoenixframework/phoenix/pull/2779">
+          Webpack 4 has been replaced by brunch
```
Comment (chrismcg) [inline installer/templates/phx_web/templates/page/index.html.eex]: Other way round right? Brunch has been replaced with Webpack 4

Comment (Gazler) [reply installer/templates/phx_web/templates/page/index.html.eex]: Definitely the other way round! Thanks.

**Comment by michalmuskala** [comment]
I wonder how useful that would be for completely new users. Maybe instead of a full list it would be better to provide a link to some upgrade guide or the change log?

**Comment by josevalim** [comment]
@michalmuskala I like this idea. Maybe a simple link under "Resources".

**Comment by Gazler** [comment]
Replaced by https://github.com/phoenixframework/phoenix/pull/2881

**Pull request closed without merging**